packages/amplication-data-service-generator/src/server/static/scripts/clean.ts

```diff
@@ -40,8 +40,7 @@ async function dropTypes(prisma: PrismaClient, types: string[]) {
 async function getTables(prisma: PrismaClient): Promise<string[]> {
   const results: Array<{
     tablename: string;
-  }> =
-    await prisma.$queryRaw`SELECT tablename from pg_tables where schemaname = 'public';`;
+  }> = await prisma.$queryRaw`SELECT tablename from pg_tables where schemaname = 'public';`;
```

Comment:
Undo this in order to fix the ci pipeline.
Locally you'd need to remove your node_modules and run npm i